.github/workflows/ci.yml

```diff
@@ -487,6 +487,49 @@ jobs:
             **/surefire-reports/TEST-*.xml
           retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
 
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: 11
+          cache: 'maven'
+      - name: Maven Install
+        run: |
+          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
+          # GIB needs to be explicitly disabled, because the gib profile enables it, but the trino-server module requires all of its dependencies to be built
+          $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -Dgib.disable -pl '!:trino-docs,!:trino-server-rpm'
+      - name: Server artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: server tarball
+          path: |
+            core/trino-server/target/*.tar.gz
+          retention-days: 1
+      - name: Product tests launcher artifact
+        uses: actions/upload-artifact@v2
```

Comment:
Is there a way to have `actions/upload-artifact` invoked once, and have shorter (=more readable) ci.yml?

Reply:
Would it be useful to use the assembly plugin to create a nice bundled up artifact with whatever's required for PTs. It would serve as both documentation and help keep external world not care as much about what is in that artifact?

Not sure if it's possible to do though, just an idea.

Reply:
Done! Looks much better.